### PR TITLE
Test Platform: Implement React-quill

### PR DESCRIPTION
**Pull request by datngo21**
Test Platform: Implemented React-quill and optimized upload image, BE: extend body limit data on payload

### CODE REVIEW

File: test-platform-ui/app/lib/apis/ApiHook.ts [hunk 1]
```diff
@@ -9,8 +9,8 @@ export enum Methods {
 }
 
 const axiosInstance = axios.create({
-  // baseURL: 'http://localhost:3000',
-  baseURL: 'https://test-platform-api.onrender.com',
+  baseURL: 'http://localhost:3000',
+  // baseURL: 'https://test-platform-api.onrender.com',
```
Comment (sts-nghianguyen) [inline test-platform-ui/app/lib/apis/ApiHook.ts]: Revert to the https instead of local

File: test-platform-ui/app/lib/editor/richtext.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import {
```
Comment (sts-nghianguyen) [inline test-platform-ui/app/lib/editor/richtext.tsx]: This should be place in the Atoms folder, lib folder should be contain the configuration only

File: test-platform-ui/app/lib/apis/ApiHook.ts (hunk 1 above)
Comment (datngo21) [reply test-platform-ui/app/lib/apis/ApiHook.ts]: Updated, thanks @sts-nghianguyen

File: test-platform-ui/app/lib/editor/richtext.tsx (hunk 2 above)
Comment (datngo21) [reply test-platform-ui/app/lib/editor/richtext.tsx]: Relocated folder, thanks @sts-nghianguyen

**Pull request merged**